rules/windows/discovery_signal_unusual_discovery_signal_proc_cmdline.toml

```
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2023/09/22"
+maturity = "production"
+min_stack_comments = "Multiple field support in the New Terms rule type was added in Elastic 8.6"
+min_stack_version = "8.6.0"
+updated_date = "2023/09/22"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule leverages alert data from various Discovery building block rules to alert on signals with unusual unique
+host.id, user.id and process.command_line entries.
+"""
+from = "now-9m"
+index = [".alerts-security.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Unusual Discovery Signal Alert with Unusual Process Command-line"
```

```suggestion
name = "Unusual Discovery Signal Alert with Unusual Process Command Line"
```

or whatever is consistent with existing usage